ansible/ext-inventory.yml

```diff
@@ -0,0 +1,65 @@
+# This playbook updates data at ooni-sysadmin.git/ext/
+---
+- hosts: localhost
+  connection: local
+  gather_facts: false
+  tasks:
+    - name: get git-root
+      command: git rev-parse --show-toplevel
+      register: extroot
+      changed_when: false
+
+    - set_fact:
+        extroot: '{{ extroot.stdout }}/ext'
+        socks5: '{{ 32767 | random(1024) }}'
+        jumphost: '{{ namecheap_jumphosts | random }}'
+    - name: ssh jumphost
+      command: ssh -D {{ socks5 }} {{ jumphost }} sleep 60
```

Comment:
I usually add the `-Nq` options to ssh jumphost commands as there is no need to execute a remote command to the jumpohost or be that verbose.